src/core/Microsoft.PowerToys.Settings.UI.Runner/Program.cs

```diff
@@ -31,6 +36,8 @@ public static void Main(string[] args)
 
                 if (args.Length >= ArgumentsQty)
                 {
+                    PowerToysPID = int.Parse(args[2]);
+
                     if (args[4] == "true")
```

Comment:
You added a new argument, is that effecting `args[4]`?

Reply:
No, I use the argument that was present before